be/src/olap/merger.cpp

```diff
@@ -65,6 +65,11 @@ OLAPStatus Merger::merge_rowsets(TabletSharedPtr tablet,
         RETURN_NOT_OK_LOG(dst_rowset_writer->add_row(row_cursor),
                           "failed to write row when merging rowsets of tablet " + tablet->full_name());
         output_rows++;
+        if (config::row_step_for_compaction_merge_log != 0 && output_rows % config::row_step_for_compaction_merge_log == 0) {
```

Comment:
There is "LOG_EVERY_N" macro for this purpose

Reply:
> There is "LOG_EVERY_N" macro for this purpose

OK.

Reply:
> There is "LOG_EVERY_N" macro for this purpose

"LOG_EVERY_N" will print a message after "LOG_EVERY_N" is executed `n` times. After test，"LOG_EVERY_N" macro is for all compaction threads，it means  a message will be printed after `n` rows merged for all compaction tasks. But we need a message after `n` rows merged for a particular compaction task.